wordpress-mobile/gutenberg-mobile
# Add gb mobile specific initial html

This replaces the default HTML demo content with just non-core blocks (at the moment that's just the `jetpack/contact-info` block).
The related PRs are:
- https://github.com/WordPress/gutenberg/pull/25844 removes these non-core blocks from Gutenberg
- https://github.com/WordPress/gutenberg/pull/25865 allows us to add core blocks back here

To test:
1. Run `npm run start:reset` from the `gutenberg-mobile` folder
2. Run `npm run core ios` or `npm run core android` to open the demo app
3. Expect to see ~~just non-core blocks in the editor (no core blocks)~~ Jetpack blocks followed by core blocks

PR submission checklist:
